datafusion/core/tests/sqllogictests/test_files/ddl.slt

```diff
@@ -296,9 +320,20 @@ SELECT * FROM my_table;
 ----
 1 2 hello
 
+# select_into_with_schema_as_values
+statement ok
+SELECT * INTO new_table FROM my_table
```

Comment:
I don't understand how this test is different than the ones above. Is the intent to insert into a table in a schema? Like `SELECT * into my_schema.new_table from my_table`?

Reply:
I think the comments have made the intent clear -- thank you